### DISCUSSION TITLE
Livewire applications don't work on Server

### DISCUSSION THREAD

**Discussion by ghost** [livewire/livewire#4575] [1. Help]
I have a question.

When I write a Laravel program with Livewire it runs great on my development machine (server start with php artisan serve). As soon as I load the program on my server the Livewire applications don't work anymore. What can be the reason for this?

Do you have any idea?

**Comment by joshhanley** [comment]
@Veritexx you will have to do some troubleshooting on your server to see what is going on. It's hard for us to say what is going on without any information.

You will need to check that all of your composer dependencies have installed correctly on the server.

Then if you browse to your production site and open up dev tools and go to the console, when you type in `window.Livewire` you should see the Livewire javascript object appear. Can you try that and let us know what happens?

Thanks!
